Attributes/ValidationAttributes/MaxFileSizeAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUkDesignSystem.Attributes.ValidationAttributes
+{
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class MaxFileSizeAttribute : ValidationAttribute
```

Comment:
To be consistent with our other attributes this would need to be something like GovUKValidateFileSizeAttribute